docs/scipp-neutron/instrument-view.ipynb

```diff
@@ -81,7 +81,7 @@
    "source": [
     "You can adjust opacities and create a cut surface to slice your data in 3D using the control buttons below the scene.\n",
     "\n",
-    "It is possible to customize the figure using the usual arguments, as well as adjusting the pixel size, e.g."
+    "It is possible to customize the figure using the usual arguments, e.g."
```

Comment:
Why was this removed? Is it documented elsewhere? I think it will still be frequently used, at least for instrument with non-square pixels.